cranelift-codegen/src/isa/registers.rs

```diff
@@ -12,25 +12,16 @@ use core::fmt;
 /// The register allocator will enforce that each register unit only gets used for one thing.
 pub type RegUnit = u16;
 
-/// A bit mask indexed by register units.
-///
-/// The size of this type is determined by the target ISA that has the most register units defined.
-/// Currently that is arm32 which has 64+16 units.
-///
-/// This type should be coordinated with meta/src/cdsl/regs.rs.
-pub type RegUnitMask = [u32; 3];
-
 /// A bit mask indexed by register classes.
 ///
 /// The size of this type is determined by the ISA with the most register classes.
-///
-/// This type should be coordinated with meta/src/cdsl/regs.rs.
 pub type RegClassMask = u32;
 
-/// Guaranteed maximum number of top-level register classes with pressure tracking in any ISA.
+/// A bit mask indexed by register units.
 ///
-/// This can be increased, but should be coordinated with meta/src/cdsl/regs.rs.
-pub const MAX_TRACKED_TOPRCS: usize = 4;
+/// The size of this type is determined by the target ISA that has the most register units defined.
+/// Currently that is arm32 which has 64+16 units.
+pub type RegUnitMask = [RegClassMask; 3];
```

Comment:
Should the 3 here be in constants too?

Reply:
Not sure, since it doesn't need to be synced between meta and codegen.

Reply:
Maybe meta can emit it as constant to match the isa with the most reg units.

Reply:
It's a nice idea: we could add the total number of regunits belonging to register classes that track register pressure, and use the maximum number in this array. Maybe something to do later yeah.